src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,31 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/add")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long carId = Long.valueOf(req.getParameter("carId"));
+        Long driverId = Long.valueOf(req.getParameter("driverId"));
```

Comment:
Use the same style 
![image](https://github.com/mate-academy/jv-web-practice/assets/20326858/26bcdbd1-0848-4c5a-9d78-93d9b3cac9d3)

```suggestion
        Long carId = Long.valueOf(req.getParameter("car_id"));
        Long driverId = Long.valueOf(req.getParameter("driver_id"));
```